setup.cfg

```diff
@@ -31,22 +31,22 @@ include_package_data = True
 packages = find:
 python_requires = >=3.6
 install_requires =
-    numpy ==1.18.1
-    pandas ==1.0.3
-    dask[complete] ==2.17.2
-    xarray ==0.15.1
-    matplotlib ==3.1.3
-    notebook >=5.5
+    numpy
+    pandas
+    dask[complete]
```

Comment:
Any idea whether we really need "complete"?

Reply:
Honestly, I'm not sure...I simply removed the pins without thinking about anything else.